src/components/workspace.tsx

```diff
@@ -73,9 +75,42 @@ export class WorkspaceComponent extends BaseComponent<IProps, {}> {
   }
 
   private renderSupports() {
+    const supports = this.getSupportsWithIndices();
     return (
       <div className="supports">
-        <span>TBD: Just In Time Supports</span>
+        <div className="supports-list">
+          {supports.map((support) => {
+            return (
+              <span
+                key={support.index}
+                onClick={this.handleToggleSupport(support.item)}
+                className={support.item.visible ? "active" : undefined}
+              >
+                {support.index}
+              </span>
+            );
+          })}
+        </div>
+      </div>
+    );
+  }
+
+  private renderVisibleSupports() {
```

Comment:
This method supports multiple active supports - will that ever be the case? I'd also clarify naming of these two methods e.g. `renderSupportIcons` and `renderSupportPrompts`